docs/admin/admin_install.md

```diff
@@ -87,19 +88,20 @@ sudo systemctl restart nautobot nautobot-worker nautobot-scheduler
-| per_feature_height | 4 | 4 | The height in inches that the overview table can be. |
+    The `enable_backup`, `enable_compliance`, `enable_intended`, `enable_sotagg` and `enable_postprocessing` will toggle inclusion of the entire component.
+
+| Key                   | Example                       | Default | Description                                                                                                                                              |
+| --------------------- | ----------------------------- | ------- | -------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| enable_backup         | True                          | True    | A boolean to represent whether or not to run backup configurations within the plugin.                                                                    |
+| enable_compliance     | True                          | True    | A boolean to represent whether or not to run the compliance process within the plugin.                                                                   |
+| enable_intended       | True                          | True    | A boolean to represent whether or not to generate intended configurations within the plugin.                                                             |
+| enable_sotagg         | True                          | True    | A boolean to represent whether or not to provide a GraphQL query per device to allow the intended configuration to provide data variables to the plugin. |
+| enable_postprocessing | True                          | True    | A boolean to represent whether or not to generate intended configurations to push, with extra processing such as secrets rendering.                      |
+| platform_slug_map     | {"cisco_wlc": "cisco_aireos"} | None    | A dictionary in which the key is the platform slug and the value is what netutils uses in any "network_os" parameter.                                    |
+| sot_agg_transposer    | "mypkg.transposer"            | None    | A string representation of a function that can post-process the graphQL data.                                                                            |
+| per_feature_bar_width | 0.15                          | 0.15    | The width of the table bar within the overview report                                                                                                    |
+| per_feature_width     | 13                            | 13      | The width in inches that the overview table can be.                                                                                                      |
+| per_feature_height    | 4                             | 4       | The height in inches that the overview table can be.                                                                                                     |
```

Comment:
Should config_postprocessing_callables be here?

Reply:
resolved in https://github.com/nautobot/nautobot-plugin-golden-config/commit/5b88c8afa091542b049f6de527ecb0d4001df50b